packages/core/src/transport/httpRequest.ts

```diff
@@ -47,6 +47,15 @@ export function createHttpRequest(endpointBuilder: EndpointBuilder, bytesLimit:
     }
   }
 
+  function isKeepAliveSupported() {
+    // Request can throw, cf https://developer.mozilla.org/en-US/docs/Web/API/Request/Request#errors
+    try {
+      return window.Request && 'keepalive' in new Request('http://a')
+    } catch {
+      return false
+    }
+  }
+
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what’s best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** using `Request('http://a')` means this block is no longer needed. Might be worth moving back to the original pattern and just adding the comment?

Reply:
Since a failure caused by this check can be quite impacting, I'd be in favor of keeping this try/catch just for undocumented / future throw or patched API.